io.github.quodlibet.QuodLibet.yaml

```diff
@@ -15,6 +15,12 @@ finish-args:
   - "--socket=pulseaudio"
   - "--filesystem=host"
 
+  # some embedded covers and the fifo is still shared through /tmp
```

Comment:
Is it not configurable? Even just setting `$TMPDIR=$XDG_CACHE_HOME` is cleaner if possible.

Reply:
We currently do a lot of unnecessary writes to /tmp that need to be fixed (and I didn't notice because its tmpfs). I'll try to fix this upstream for the next version.